smart-zeppelin/zeppelin-server/src/main/java/org/smartdata/server/rest/RuleRestApi.java

```diff
@@ -41,7 +42,7 @@
 @Path("/rules")
 @Produces("application/json")
 public class RuleRestApi {
-  private SmartEngine smartEngine;
+  private final SmartEngine smartEngine;
   private static final Logger logger =
```

Comment:
Move the constant to the very top, so it appears above the declaration of the non-static field.